### DISCUSSION TITLE
Can we choose another field for card titles?

### DISCUSSION THREAD

**Discussion by MelGrubb** [marcusolsson/obsidian-projects#767] [Help]
I have a pattern established where all assets related to a particular task are gathered together in a folder together. I'm using the Folder Note plugin so that clicking on that folder opens the top-level note, which I normally call "ReadMe". These top-level notes are the same items I want to be represented in the board view, since they represent the task itself. I can't very well have a big wall of cards all called "ReadMe", though, so I switched my Folder Note settings to name the file the same as the folder. Now I have cards titled properly, but I have to rename all of my other non-task ReadMe files because Folder Note can only be in one mode at a time.

Is there any way we could just choose a property to use as the card titles? I'd be more than happy to add a "Title" property to those documents that represent kanban cards. I can already select additional fields to display ON the card, but I can't find a way to change the title of the card itself. If I've missed something in the docs, please let me know. If not, what would the effort be to add this in? Maybe a simple check for a property called "Title". If it exists, use it, and fall back on the file name, otherwise, like it's doing now.

**Comment by Acylation** [comment]
Thanks for the suggestion. This would be a new feature. I will track this together with #367 as a new design for Board view.
